scripts/source-maintenance.sh

```diff
@@ -66,7 +66,8 @@ for FILENAME in `git ls-files`; do
 	#
 	# Code Style formatting maintenance
 	#
-        if test "${ASVER}"; then
+	awk -f ./scripts/maintenance.awk "${FILENAME}" >"${FILENAME}.awkbak" && mv "${FILENAME}.awkbak" "${FILENAME}"
```

Comment:
Please also avoid using the backup-associated `bak` extension for newly created files. Something like `.next` or `.future` would be a lot more clear/natural.

Reply:
Done using .new